### PR TITLE
appropriately check for missing extremes data

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: inst/extdata/extremes/extremes-example.json [hunk 1]
```diff
@@ -112,8 +112,7 @@
     "locationNumber": "06933500",
     "locationLon": "-91.9773333333333",
     "dvUnit": "ft^3/s",
-    "startDate": "2014-12-16T00:00:00-06:00",
-    "isInverted": false
```
Comment (mhines-usgs) [inline inst/extdata/extremes/extremes-example.json:116]: is this not needed? (isInverted?) @kmschoep-usgs

File: R/sig-extremes.R [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #'@importFrom rmarkdown render
 #'@examples
 #'library(jsonlite)
-#'data <- fromJSON(system.file('extdata','extremes','extremes-example.json',package = 'repgen'))
```
Comment (mhines-usgs) [inline R/sig-extremes.R:10]: you could leave the original file (unless it's no longer a valid dataset to test with) and just have two examples here?

File: inst/extdata/extremes/extremes-example.json (hunk 1 above)
Comment (kmschoep-usgs) [inline inst/extdata/extremes/extremes-example.json:116]: It was a duplicate isInverted attribute

Comment (mhines-usgs) [inline inst/extdata/extremes/extremes-example.json:116]: :+1:

**Pull request merged**